### PR TITLE
Tma 702 add all new events

### PR DESCRIPTION
**Pull request by PeterBrownGDS**

### CODE REVIEW

**Review by zaydgds — DISMISSED** [review]
all good :)

File: tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java [hunk 1]
```diff
@@ -101,6 +101,7 @@ public void checkEndpointJSONIsAvailable(String fileName, String account, DataTa
     @When("the {string} lambda is invoked")
     public void invokeAccountsLambda(String account) {
         String functionName = "EventProcessorFunction-" + account;
+//        String functionName = "EventProcessorFunction-App";
```
Comment (dthampi) [inline tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java:104]: Think we can delete this.

Comment (PeterBrownGDS) [reply tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java:104]: Removed line 104

**Review by dthampi — APPROVED** [review]
LGTM

**Pull request merged**